docs/_sidebar.json

```diff
@@ -21,7 +21,8 @@
             { "text": "Debugging QMK", "link": "/faq_debug" },
             { "text": "Keymap FAQ", "link": "/faq_keymap" },
             { "text": "Squeezing Space from AVR", "link": "/squeezing_avr" },
-            { "text": "Glossary", "link": "/reference_glossary" }
+            { "text": "Glossary", "link": "/reference_glossary" },
+            { "text": "Deprecation Policy", "link": "/support_deprecation_policy" }
```

Comment:
Would this be better under the "Breaking Changes" section?

Reply:
Done.